cmd/createorupdate/createorupdate.go

```diff
@@ -29,7 +28,9 @@ var logLevel = flag.String("loglevel", "Debug", "valid values are Debug, Info, W
 // createOrUpdate simulates the RP
 func createOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCluster, entry *logrus.Entry) (*v20180930preview.OpenShiftManagedCluster, error) {
 	// instantiate the plugin
-	p := plugin.NewPlugin(entry, os.Getenv("SYNC_IMAGE"))
+	var config = acsapi.PluginConfig{SyncImage: os.Getenv("SYNC_IMAGE"),
```

Comment:
Change this to be an input to `createOrUpdate`. It will be useful once I move `createOrUpdate` in its own library and reuse in our e2e framework.